hi-ml-histopathology/src/histopathology/utils/output_utils.py

```diff
@@ -357,7 +356,6 @@ def save_test_outputs(self, epoch_results: EpochResultsType, is_global_rank_zero
         # All DDP processes must reach this point to allow synchronising epoch results
         gathered_epoch_results = gather_results(epoch_results)
         if PlotOption.TOP_BOTTOM_TILES in self.test_plots_handler.plot_options and self.tiles_selector:
-            logging.info("Selecting tiles ...")
```

Comment:
any specific reason to delete the logging?

Reply:
Not really, Anton mentioned earlier that we have a lot of verbosity here and there. I grouped everything in a single log in plots_utils.py line 194.